src/index.html

```diff
@@ -10,13 +24,95 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Mate academy</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="javascript:void(0)"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="javascript:void(0)"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="javascript:void(0)"
```

Comment:
Using `javascript:void(0)` in the `href` attribute is not recommended as it can cause accessibility issues and is not SEO-friendly. Consider using `#` or a valid URL instead.